### ISSUE TITLE
Backdrop rebranding: update administrative interface elements

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#201]
We love Seven, but a big part of creating a new CMS is creating our own brand. We've been working on our new brand over in the <a href="http://backdrop-ops.github.io/styleguide">styleguide</a> and now it's time to get these changes into Backdrop itself. Since 1.0 has been released already, we have have to move forward carefully:

1) Make some style changes directly to core modules (system, etc). This is only possible where the markup does not need to change and the changes can all be accomplished with CSS alone. This should be done (where possible) for discrete elements like status messages, tabs, drop-buttons, etc.

2) Make some changes to the seven theme instead of to core modules. We will need to do this for things like the font & background header color - things that were previously defined by the Seven theme. We will also need to do this where the markup needs to change to accommodate our new design changes in core elements.

Changes we can make to core (no markup changes necessary):
- [ ] Dropbuttons
- [ ] Tables
- [ ] ??

Changes we need to make to Seven:
- [ ] Header background color
- [ ] Font face (Open Sans)
- [ ] Heading & font sizes
- [ ] ??

Areas where we may still need some design/animation help
- [ ] Throbber
- [ ] Progress Bar
- [ ] ??

Related Issue: https://github.com/backdrop/backdrop-issues/issues/139

**Comment by dgoutam** [comment]
@jenlampton "Yep, it's a reskinning exercise, exactly" if it is so by choice then how backdrop will rebrand it, a bit confused.

**Comment by jenlampton** [comment]
We're making a bunch of progress over in the style guide this week: http://backdrop-ops.github.io/styleguide

**Comment by BWPanda** [comment]
Possible new name for Backdrop admin theme: Administrate
Or, since it's the successor to Seven: Administr8 ;)

**Comment by klonos** [comment]
@jenlampton: Hey Jen, I was taking a look at the [styleguide](https://backdrop-ops.github.io/styleguide) you mentioned a few weeks ago in your last comment here. I realize that it must be WIP, but I've spotted a few things. Where should I file the issues? Is this the place?

Also, just to be clear, that is the styleguide for our future admin theme or for the new them for bd.org?

**Comment by jenlampton** [comment]
@klonos That is the styleguide for backdrop's admin theme (rebranding so it looks less like Drupal 7). It's still a work-in-progress over there, but we should be getting close. The best place to file issues would be directly in the styleguide's issue queue: https://github.com/backdrop-ops/styleguide/issues (I just turned it on - sorry about that oversight).

**Comment by jenlampton** [comment]
Update based on a conversation we had in the Backdrop Design fortnightly meeting last week:

Since we have already shipped 1.0 we need to decide on the best way to get our new branding into core for the 1.1 release.  

We discussed adding a new administration theme, and changing the default for new Backdrop installs starting after 1.1.  We decided against that since adding more themes to core that are not front-end themes seems like a bad idea.  If we're going to add a new theme in 1.0 it should be a front-end theme not an administration theme.

So that leaves us with the following path forward: 

1) Make the style changes directly to core modules (system, etc).  This is only possible where the markup does not need to change and the changes can all be accomplished with CSS alone. This should be done (where possible) for discrete elements like status messages, tabs, drop-buttons, etc.

2) Make the changes to the seven theme instead of to core. We will need to do this for things like the fonts & background header color - things that were previously defined by the Seven theme. We will also need to do this where the markup needs to change to accommodate new design changes in core elements (status messages, tabs, drop-buttons, etc)

**Comment by jenlampton** [comment]
@ronnienorwood I know you have been making a bunch of progress on this in your PRs, we should decide how to best divide that work into actionable issues, and assign some to you so you get unblocked and can continue making good progress :) 

I believe @wruvalcaba will also be joining us on this task, let's make sure you two don't accidentally start working on the same things.

**Comment by wesruv** [comment]
Looking at the current style guide and the designs it looks like this is going to be a mix of:
* implement existing design for some elements (e.g. admin menu didn't match design)
* design and implement styles for elements that haven't been accounted for yet, inside of the desired look and feel (e.g. a lot of form elements, throbber, etc.)

Does that sounds right?

**Comment by jenlampton** [comment]
yes, generally.  The one exception is that Admin Bar is actually correct in core, the styleguide just doesn't have the most recent code.  If you can ignore the admin_bar styles in the styleguide, and focus on everything else you should be golden :)

**Comment by ronnienorwood** [comment]
@jenlampton  Plans are good.  For the pull requests, I think we definitely should have one PR for core and another for the admin theme at the very least.  I would lean toward having one PR per element here for better organization even though I did lump all of the changes to several elements together when I worked on the styleguide.

Do you have a handle on who all is working on the design area - @wesruv, @dariusgarza, myself, and who else ??

**Comment by wesruv** [comment]
@jenlampton @ronnienorwood @dariusgarza 
I vote for a PR per related sets of elements.. so less than one per element but more than 2 :smile: 

I think a meeting with some screensharing would help me wrap my head around the work and collaboration. Look like there are two on Thursday. I have hangout info for the Backdrop weekly and **no info** for the Backdrop Design meeting. Don't know if one or neither of those is more appropriate to get us going on this? I'd be interested in being at least a fly on the wall for both.

In the mean time I'm going to dink around with making an AJAX in progress indicator (aka throbber)

**Comment by ronnienorwood** [comment]
@wesruv  Sounds like a reasonable compromise on the PRs.

With you...waiting for more specifics on the design meeting.

**Comment by wesruv** [comment]
@ronnienorwood so it sounds like the easiest way to join the Design meeting is to monitor the twitter feed (@backdropcms) for a link to the agenda when the meeting is about to start. It'll have a link to the hangout (which is unfortunately unique, every time)

**Comment by ronnienorwood** [comment]
Thanks @wesruv.  Missed the meeting so I will watch the video.

**Comment by jenlampton** [comment]
I just added a "video call" to each of our weekly meetings. I had trouble with this in the past, so we'll see if it actually works this time. Fingers crossed!

**Comment by quicksketch** [comment]
Looks like this is now a duplicate of https://github.com/backdrop/backdrop-issues/issues/736. There's no need to have both of these issues.
